pinery-lims-miso/src/main/java/ca/on/oicr/pinery/lims/miso/MisoClient.java

```diff
@@ -318,8 +317,17 @@
       "        INNER JOIN Indices ON Indices.indexId = Library_Index.index_indexId\n" + 
       "                WHERE position = 2\n" + 
       "        ) bc2 ON bc2.library_libraryId = l.libraryId\n" + 
-      "LEFT JOIN BoxPosition pos ON pos.boxPositionId = l.boxPositionId\n" + 
+      "LEFT JOIN BoxPosition pos ON pos.targetId = l.libraryId\n" + 
+      "        AND pos.targetType = 'L'\n" +
```

Comment:
I thought I had made that request, and I thought it had been done, but then there were S's and L's :confused: